src/cards/target_column.ts

```diff
@@ -31,7 +31,7 @@ const spell: Spell = {
     allowNonUnitTarget: true,
     effect: async (state, card, quantity, underworld, prediction, outOfRange) => {
       const depth = (range + state.aggregator.radius) * (0.5 + 0.5 * quantity);
-      const width = (baseWidth + state.aggregator.radius) * (0.5 + 0.5 * quantity);
+      const width = baseWidth * Math.pow(2, Math.min(quantity, 4)) / 2;
```

Comment:
A comment would be helpful here so I don't have to mentally process the math to understand how or why it scales this way